includes/tracks/class-wc-tracks-client.php

```diff
@@ -98,7 +136,7 @@ public static function get_anon_id() {
 		if ( ! isset( $anon_id ) ) {
 
 			// Did the browser send us a cookie?
-			if ( isset( $_COOKIE['tk_ai'] ) && preg_match( '#^[A-Za-z0-9+/=]{24}$#', $_COOKIE['tk_ai'] ) ) {
+			if ( isset( $_COOKIE['tk_ai'] ) ) {
```

Comment:
The regex removed here always resulted in `false` for id's generated here and those created by Jetpack:

```
jetpack:oKXKpD6dLQTOxXfdrwjxD2l1
```
this resulted in a new id created every time.

Reply:
If this is the format, we probably can run `sanitize_text_field()` to sanitize it.